lib/shared/cross_posts.dart

```diff
@@ -47,157 +52,117 @@ class _CrossPostsState extends State<CrossPosts> with SingleTickerProviderStateM
-                                : const Icon(Icons.arrow_drop_down_rounded),
-                      ],
-                    ),
-                    // This Row widget exists purely so that we can get an InkWell on the community link.
-                    // However, the text is insvisible because we actually want the RichText to manage the text,
-                    // including overflow.
-                    Padding(
-                      padding: const EdgeInsets.only(top: 5),
-                      child: Row(
-                        crossAxisAlignment: CrossAxisAlignment.end,
-                        children: [
-                          Text(
-                            widget.isNewPost == true ? l10n.alreadyPostedTo : l10n.crossPostedTo,
-                            style: crossPostTextStyle?.copyWith(color: Colors.transparent),
-                          ),
-                          InkWell(
-                            borderRadius: BorderRadius.circular(5),
-                            onTap: () async => navigateToPost(context, postViewMedia: (await parsePostViews([widget.crossPosts[0]])).first),
-                            child: Text(
-                              ' ${generateCommunityFullName(context, widget.crossPosts[0].community.name, fetchInstanceNameFromUrl(widget.crossPosts[0].community.actorId))} ',
-                              style: crossPostLinkTextStyle?.copyWith(color: Colors.transparent),
-                            ),
-                          ),
-                        ],
-                      ),
-                    ),
-                  ],
-                ),
-              ),
-            ),
-            AnimatedSwitcher(
-              duration: const Duration(milliseconds: 200),
-              switchInCurve: Curves.easeInOut,
-              switchOutCurve: Curves.easeInOut,
-              transitionBuilder: (Widget child, Animation<double> animation) {
-                return SizeTransition(
-                  sizeFactor: animation,
-                  child: SlideTransition(position: _offsetAnimation, child: child),
-                );
-              },
-              child: _areCrossPostsExpanded
-                  ? Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        ListView.builder(
-                          shrinkWrap: true,
-                          physics: const NeverScrollableScrollPhysics(),
-                          itemBuilder: (context, index) {
-                            return Padding(
-                              padding: const EdgeInsets.only(left: 5),
+                    ListView.builder(
+                      shrinkWrap: true,
+                      physics: const NeverScrollableScrollPhysics(),
+                      itemBuilder: (context, index) {
+                        return Column(
+                          children: [
+                            /*const Divider(),*/

```

Comment:
I would remove this commented code to keep the code base clean

Reply:
Yep, accidental.